src/Native/Runtime/coreclr/gcinfotypes.h

```diff
@@ -323,7 +323,6 @@ inline const char *ReturnKindToString(ReturnKind returnKind)
 #ifdef _TARGET_X86_
 
 #include <stdlib.h>     // For memcmp()
-#include "bitvector.h"  // for ptrArgTP
```

Comment:
This should not be necessary with the other change. (And we would like to keep this file in sync with CoreCLR - removing this is likely going to break CoreCLR build.)